.github/actions/setup-azurite/action.yml

```
@@ -0,0 +1,20 @@
+﻿name: Set up Azurite
+description: >
+  Sets up the Azurite Azure Blob Storage Emulator
```

I haven't seen Auirite referred to as "Azure Blob Storage Emulator" yet. In fact it supports blob, queue and table storage types. If you've accidentally used title case, you don't have to.

```suggestion
  Sets up the Azurite for local Azure Blob Storage emulation
```